commands-yml/commands/template.yml

```diff
@@ -0,0 +1,99 @@
+---
+name: 'Template'
+
+description:
+  >
+    An indepth description
+    of what this command does
+example_usage:
+  java:
+    |
+      MobileElement el = driver.findElementByAccessibilityId("SomeId");
+      el.click();
+  python:
+    |
+      el = self.driver.find_element_by_accessibility_id('SomeId')
+      el.click();
+  javascript_wd:
+    |
+      await driver.status();
+  javascript_wdio:
+    |
+      await driver.status();
+  ruby:
+    |
+      # Ruby Code here
+  php:
+    |
+      # PHP Code here
+  csharp:
+    |
+      // CSharp Code here
+
+client_docs:
+  java: "https://seleniumhq.github.io/selenium/docs/api/java/org/openqa/selenium/WebElement.html#click--"
+  python: "https://github.com/appium/python-client"
+  javascript_wdio: "http://webdriver.io/index.html"
+  javascript_wd: "https://github.com/admc/wd/releases"
+  ruby: "https://github.com/appium/ruby_lib/releases/latest"
+  php: "https://github.com/appium/php-client/releases/latest-"
+  csharp: "https://github.com/appium/appium" # TODO
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest:
+      platform_min: '1.2'
+      platform_max: '9.3'
```

Comment:
i know this is the template so we're showing everything, but is it possible to omit mins and maxes and have the renderer do the right thing?

Reply:
Yeah. I wrote a handlebars helper that covers that.